### ISSUE TITLE
Cask is not configured

### ISSUE THREAD

**Issue by R0dri** [fniephaus/alfred-homebrew#39]
After the latest PR #36 error at configuration of cask appears:
![image](https://user-images.githubusercontent.com/14064394/67966874-031aac00-fbdb-11e9-83a3-fa004bb138b9.png)

It looks like there is a new conflicting package:
> Seems like there is now a font-sudo Cask, so this check breaks cask install:
> 
> if 'sudo' in result:
>     return 'Config'
> I guess this check is there in case you changed HOMEBREW_CASK_OPTS? Maybe there's a better way to check for that?
by @ErikMinekus 

This is a not so elegant solution I made. Probably breaks the purpose of the if in the first place:

```
 if 'sudo' in result:
     if 'font-sudo' not in result:
          return 'Config'
```

**Comment by R0dri** [comment]
That is true, cask does come with brew. So if brew.py has an infrastructure check then this if is just a redundancy.

**Comment by TheSecEng** [comment]
> I wonder if this check and infrastructure is still needed at all...if I understand correctly, brew ships with cask out-of-the-box, right?

This is what I am testing and researching now. It appears to be no longer required. I am testing on fresh installs and 100 ~ package installs.

**Comment by TheSecEng** [comment]
Fixed in https://github.com/fniephaus/alfred-homebrew/pull/40

**Comment by fniephaus** [comment]
Should be fixed in [`v5.0`](https://github.com/fniephaus/alfred-homebrew/releases/tag/v5.0).
